gobblin-runtime/src/main/java/gobblin/runtime/instance/StandardGobblinInstanceDriver.java

```diff
@@ -417,6 +418,11 @@ public boolean isInstrumentationEnabled() {
 
     public List<GobblinInstancePluginFactory> getPlugins() {
       List<GobblinInstancePluginFactory> res = new ArrayList<>(getDefaultPlugins());
+      // By default email notification plugin is added.
```

Comment:
Yes we can add it there but one need's to whitelist email notification. This was @chavdar 's suggestion.